components/Alert/Alert.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Button from '../Button';
+import Colors from '../Colors';
+import Icon from '../Icon';
+
+const BORDER_SIZE = '1.5px';
+const DEFAULT_SPACING = '16px';
+const WRAPPER_SPACING = '11px';
+
+const Content = styled.div`
+  align-items: start;
+  display: flex;
+`;
+
+const AlertIcon = styled(Icon)``;
+
+const CloseButton = styled(Button.Icon).attrs({
+  icon: 'close',
+})`
+  height: auto;
+  width: auto;
+  margin: 0 0 0 ${DEFAULT_SPACING};
+  padding: 0;
+  opacity: 0.8;
+  transition: opacity 0.4s ease;
+
+  :hover {
+    background: none;
+    opacity: 1;
+  }
+`;
+
+const getStylesBySkin = skin => {
+  const colorSchema = skin.toUpperCase();
+
+  return `
+    color: ${Colors[colorSchema][900]};
+    background-color: ${Colors[colorSchema][200]};
+    border: ${BORDER_SIZE} solid ${Colors[colorSchema][500]};
+
+    ${Content} ${AlertIcon} {
+      color: ${Colors[colorSchema][500]};
+      margin-right: ${DEFAULT_SPACING};
+    }
+
+    ${Content} ${CloseButton} {
+      color: ${Colors[colorSchema][500]};
+    }
+  `;
+};
+
+const Wrapper = styled.div`
+  border-radius: 8px;
+  box-sizing: border-box;
+  padding: ${WRAPPER_SPACING} ${DEFAULT_SPACING};
+
+  ${({ skin }) => getStylesBySkin(skin)}
+`;
+
+CloseButton.displayName = 'CloseButton';
+
+const Alert = ({ icon, children, onClose, ...rest }) => (
+  <Wrapper {...rest}>
+    <Content onClose={onClose}>
```

Comment:
what do you think about put a aria `role="alert"` here to assistive technologies?

https://developer.paciellogroup.com/blog/2012/06/html5-accessibility-chops-aria-rolealert-browser-support/

Reply:
I think it's great! consider it done!